docs/_docs/analysis-types/known-vulnerabilities.md

```diff
@@ -8,11 +8,13 @@ order: 1
 Dependency-Track integrates with multiple sources of vulnerability intelligence to identify components with known 
 vulnerabilities. The platform employs several methods of vulnerability identification including:
 
-| Analyzer  | Description |
-| ----------|-------------|
-| Internal  | Identifies vulnerable components from an internal directory of vulnerable software|
-| OSS Index | OSS Index is a service provided by Sonatype which identifies vulnerabilities in third-party components|
-| VulnDB    | VulnDB is a commercial service which identifies vulnerabilities in third-party components|
+| Analyzer  | Description                                                                                            |
+|-----------|--------------------------------------------------------------------------------------------------------|
+| Internal  | Identifies vulnerable components from an internal directory of vulnerable software                     |
+| OSS Index | OSS Index is a service provided by Sonatype which identifies vulnerabilities in third-party components |
+| VulnDB    | VulnDB is a commercial service which identifies vulnerabilities in third-party components              |
+| Snyk      | Snyk is an open source service which identifies vulnerabilities in third-party components                                                       |
```

Comment:
> Snyk is an open source service

Is this accurate? Isn't Snyk a proprietary / commercial service?

Reply:
Right I was not sure, will update it to commercial.